AGENTS.md

```
@@ -0,0 +1,115 @@
+| PostgreSQL | double quotes `"` | `"schema"."table"` |
+| Redshift | double quotes `"` | `"schema"."table"` |
+| ClickHouse | double quotes `"` | `"database"."table"` |
+| Spark | backticks `` ` `` | `` `database`.`table` `` |
+| Hive | backticks `` ` `` | `` `database`.`table` `` |
+| ClickZetta | double quotes `"` | `"schema"."table"` |
+
+## Testing
+
+### Test Structure Per Adapter
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)